packages/arb-util/protocol/timebounds.go

```diff
@@ -31,6 +31,15 @@ type TimeBounds struct {
 	UpperBoundTimestamp *big.Int
 }
 
+func NewRandomTimeBounds() *TimeBounds {
+	return &TimeBounds{
+		LowerBoundBlock:     common.NewTimeBlocks(common.RandBigInt()),
```

Comment:
Seems like this should be written to avoid having lowerBound > upperBound.

Reply:
Good call. That definitely makes sense and I've updated this. There's probably more validity checks we can do for the randomly generated data though I think in general it won't be an issue until we have tests that fail due to data correctness issues so I'm not too worried in about this in general.